connectors/sql-stored-connector/src/main/java/io/syndesis/connector/SqlStoredConnectorVerifierExtension.java

```diff
@@ -0,0 +1,108 @@
+
+    public SqlStoredConnectorVerifierExtension() {
+        super("sql-stored-connector");
+    }
+
+    public SqlStoredConnectorVerifierExtension(String scheme) {
+        super(scheme);
+    }
+
+    // *********************************
+    // Parameters validation
+    // *********************************
+
+    @Override
+    protected Result verifyParameters(Map<String, Object> parameters) {
+        ResultBuilder builder = ResultBuilder.withStatusAndScope(Result.Status.OK, Scope.PARAMETERS)
+            .error(ResultErrorHelper.requiresOption("driver", parameters))
+            .error(ResultErrorHelper.requiresOption("url", parameters))
+            .error(ResultErrorHelper.requiresOption("user", parameters))
+            .error(ResultErrorHelper.requiresOption("password", parameters));
+
+        
+        if (builder.build().getErrors().isEmpty() && parameters.containsKey("driver")) {
+            try {
+                Class.forName(parameters.get("driver").toString());
+            } catch (Exception e) {
+                Enumeration<Driver> drivers = DriverManager.getDrivers();
+                StringBuffer supportedDrivers = new StringBuffer("[");
+                while (drivers.hasMoreElements()) {
+                    String driverName = drivers.nextElement().getClass().getName();
+                    if (supportedDrivers.length() > 1 ) {
+                        supportedDrivers.append(", ");
+                    }
+                    supportedDrivers.append(driverName);
+                }
+                supportedDrivers.append("]. ");
+                String msgPrefix = "Unsupported Database, could not find Driver with class ";
+                String msgPostfix = ". Drivers on classpath are: " + supportedDrivers;
+                builder.error(ResultErrorBuilder.withCodeAndDescription(
+                        VerificationError.StandardCode.UNSUPPORTED, 
+                            msgPrefix + parameters.get("driver").toString() + msgPostfix).build()).build();
+            }
+        }
+        return builder.build();
+       
+    }
+
+    // *********************************
+    // Connectivity validation
+    // *********************************
+
+    @Override
+    protected Result verifyConnectivity(Map<String, Object> parameters) {
+        return ResultBuilder.withStatusAndScope(Result.Status.OK, Scope.CONNECTIVITY)
+            .error(parameters, this::verifyCredentials)
+            .build();
+    }
+
+    private void verifyCredentials(ResultBuilder builder, Map<String, Object> parameters) {
+        try {
+            Class.forName(parameters.get("driver").toString());
+            
+            Connection connection = DriverManager.getConnection(
+                    parameters.get("url").toString(), 
+                    parameters.get("user").toString(), 
+                    parameters.get("password").toString());
+            
+            if (connection == null) {
```

Comment:
Not sure this can happen according to the [docs](http://docs.oracle.com/javase/8/docs/api/java/sql/DriverManager.html#getConnection-java.lang.String-java.lang.String-java.lang.String-) it should not return `null`.